pkg/scheduler/core/generic_scheduler.go

```diff
@@ -373,7 +371,7 @@ func (g *genericScheduler) prioritizeNodes(
 
 	if klog.V(10).Enabled() {
 		for plugin, nodeScoreList := range scoresMap {
-			klog.Infof("Plugin %s scores on %v/%v => %v", plugin, pod.Namespace, pod.Name, nodeScoreList)
+			klog.InfoS("Plugin scores on pod,", "plugin", plugin, "pod", klog.KObj(pod), "node score list", nodeScoreList)
```

Comment:
Use active voice: `Plugin scored nodes for pod`

camel case for argument names https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments

Reply:
Also, I think we should be doing one log line per node.

This only runs if level 10 is on, so it's fine.

Reply:
Thanks for your advice.